eventmesh-admin/eventmesh-admin-rocketmq/src/main/java/org/apache/eventmesh/admin/rocketmq/controller/AdminController.java

```diff
@@ -28,14 +28,30 @@
 
 import lombok.extern.slf4j.Slf4j;
 
+/**
+ * This class is responsible for managing the admin module.
+ * <p>
```

Comment:
Java 11 doc seems to not support the "<>" symbol, which means that CI will fail.

Reply:
Ignore this comment. I mixed it up with a previous issue I encountered.

Reply:
From [Oracle Java 11 Tools Reference](https://docs.oracle.com/en/java/javase/11/tools/javadoc.html), it seems that Java 11 still supports HTML tags.

Reply:
Thank you for your explanation. 
This is my previous modification to fix doc error reported by CI.
https://github.com/apache/eventmesh/pull/3988/commits/290bb93741675a525aaf97b9ce33a5c0bb85e29d

Reply:
Your example was very useful to me. There is a grammar mistake in your PR's changes that I was trying to fix earlier and I will mark it in that PR.